game.js

```diff
@@ -159,6 +157,19 @@ class Game {
     return this.getStep();
   }
 
+  /**
+   * 重新開始
+   */
+  restart() {
+    for (let bid = 0; bid < this.app.brickCount; bid++) {
+      this.bricks[bid].facePattern = Object.assign({}, this.bricks[bid].facePatternOriginal);
```

Comment:
Why not use ES6 syntax
```suggestion
      this.bricks[bid].facePattern = { ...this.bricks[bid].facePatternOriginal };
```